src/main/java/com/sendgrid/Client.java

```diff
@@ -10,7 +10,7 @@
 import java.util.Map;
 
 import org.apache.http.Header;
-import org.apache.http.StatusLine;
+import org.apache.http.HttpMessage;
```

Comment:
What's the benefit of this library over StatusLine?

Reply:
StatusLine import was removed because it wasn't needed.
HttpMessage import was added for new writeContentTypeIfNeeded() method.